site/en/docs/extensions/migrating/rhc/index.md

```diff
@@ -0,0 +1,178 @@
+
+
+## I was told my extension has RHC - what's going on?
+
+If your extension was rejected during review with a [Blue Argon][blue-argon] error, then our reviewers believe that your extension is currently using Remotely Hosted Code. This is usually the result of an extension trying to add a script tag  with a remote resource (i.e. from the open web, rather than the files included in the extension), or <code>[fetch][fetch]</code>ing a resource to execute directly.
+
+
+## How to spot RHC
+
+Spotting RHC isn't particularly difficult once you know what to look for. A great first option would be to check for the strings "http://" or "https://" in your project. If you have an RHC violation, then you would likely be able to locate them by finding that. If you have a full build system, or use dependencies from [npm][npm] or other third party sources, make sure you are searching the _compiled_ version of the code, since that is what is being evaluated by the store. If you are still unable to find the problem, then the next step would be to contact [One Stop Support][oss] They will be able to outline the specific violations, and what is needed to get the extension published as soon as possible.
+
+
+## What to do if a library is requesting the code
+
+Regardless of where the code comes from, it is not allowed to have RHC. This includes code you didn't author, but just happen to use as a dependency in your project. [Some developers][firebase-rhc-issue] using [Firebase][firebase] had this frustration when remote code was being included for use in [Firebase Auth][firebase-auth]. Even though this was a first party (i.e. Google owned) library - no exception is given for RHC. That code needs to be updated to either remove the RHC, or be able to be configured to not include it in the first place. If you end up hitting an issue where it isn't _your _code that is loading RHC, but a library that you are using, then the best course of action is to contact the library's author. Let them know that this is happening, and ask for either a workaround or code updates to remove it.
+
+
+## What if you can't wait for a library update
+
+Some libraries will ship an update almost immediately after being notified, but others may be abandoned or take time to address the issue. Depending on _what_ is happening in the specific violation, you may not need to wait for them to move to be unblocked and complete a successful review. There are a number of options available to get back up and running quickly.
+
+
+
+### Audit the code
+
+Are you _certain_ that the code that is causing the request is needed? If it can just be deleted, or a library that is causing it can be removed - then we have an easy fix -  delete the code, and the job is done.
+
+Alternatively, is there another library that offers the same functionality? Try checking [npmjs.org](npmjs.org), github, or other sites for other options that fufill the same usecases.
+
+### [Tree Shaking][tree-shaking]
```

Comment:
I don't think it looks great when headings have external links. Could we rewrite the first line of the paragraph to include "Tree shaking" and put the link there instead?

e.g: [Tree Shaking][tree-shaking] allows tooling to automatically delete unused code that may be causing an RHC violation if it isn't being used...

Same feedback for the other headings with this.

Reply:
done